### ISSUE TITLE
Possibility of insert "build now" in the view

### ISSUE THREAD

**Issue by DapperRodrigo** [jenkinsci/build-monitor-plugin#16]
Could you please inserting the "build now" in the view
##

**Comment by drekbour** [comment]
I have BM on a separate screen and often have to click through to investigate or re-run a broken build. I'm still not _sure_ if I want this feature.
* If using BFA plugin, it's the "indications" I'd open not the unformatted console output
* If using parameterised builds, that Build button actually opens another page rather than kicking off a build.
* When investigating broken builds I often want to look at the first failing build not the latest to see the Changes and culprits so I can "blame" the build (Claim plugin) rather than fixing myself.

Basically I'm not sure how general this is compared to simply middle-clicking to open a new tab. I guess simpler usages might benefit.
